server/sonar-server/src/test/java/org/sonar/server/test/ws/TestsListActionTest.java

```diff
@@ -40,10 +41,12 @@
 import org.sonar.server.test.index.TestIndexDefinition;
 import org.sonar.server.tester.UserSessionRule;
 import org.sonar.server.ws.WsTester;
+import org.sonar.test.DbTests;
 
 import java.util.Arrays;
 import java.util.List;
 
+@Category(DbTests.class)
```

Comment:
Just a detail, this file could probably have been modified in a separate commit (or even branch).